app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,13 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.all
+    @user = User.all
+    @users = User.find(params[:user_id])
+    @posts = @users.posts
   end
 
   def show
     @post = Post.find(params[:id])
+    # @comments = @post.comments.includes(:user)
+    # @users = @comments.map(&:user).uniq
```

Comment:
**[Optional]**

- [ ] It would be nice if you remove this commented piece of code to make your codebase clean and to follow Ruby best practices 😊